java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -4801,29 +4799,29 @@ public T sortReadOnly(@NonNull String key) {
      *     </code>.
      */
     public T sortStore(@NonNull String key, @NonNull String destination) {
-        ArgsArray commandArgs = buildArgs(new String[] {key, STORE_COMMAND_STRING, destination});
+        String[] commandArgs = buildArgs(new String[] {key, STORE_COMMAND_STRING, destination});
         protobufTransaction.addCommands(buildCommand(Sort, commandArgs));
         return getThis();
     }
 
     /** Build protobuf {@link Command} object for given command and arguments. */
     protected Command buildCommand(RequestType requestType) {
-        return buildCommand(requestType, buildArgs());
+        return Command.newBuilder()
+                .setRequestType(requestType)
+                .setArgsArray(Command.ArgsArray.newBuilder().build())
+                .build();
     }
 
     /** Build protobuf {@link Command} object for given command and arguments. */
-    protected Command buildCommand(RequestType requestType, ArgsArray args) {
-        return Command.newBuilder().setRequestType(requestType).setArgsArray(args).build();
+    protected Command buildCommand(RequestType requestType, String... args) {
+        final Command.Builder builder = Command.newBuilder();
+        builder.setRequestType(requestType);
+        CommandManager.populateCommandWithArgs(args, builder);
+        return builder.build();
     }
 
-    /** Build protobuf {@link ArgsArray} object for given arguments. */
-    protected ArgsArray buildArgs(String... stringArgs) {
-        ArgsArray.Builder commandArgs = ArgsArray.newBuilder();
-
-        for (String string : stringArgs) {
-            commandArgs.addArgs(ByteString.copyFromUtf8(string));
-        }
-
-        return commandArgs.build();
+    /** Dummy function for taking a series of String parameters and returning an String array */
+    private static String[] buildArgs(String... args) {
```

Comment:
I replaced the existing buildArgs() function with this dummy one that takes it's inputs and returns a String[] to make it easier to mass-update all 200+ functions.

It would be better to just inline parameters but that turned out to be more manual and probably harder to merge.